lib/tests/streamlit/watcher/local_sources_watcher_test.py

```diff
@@ -69,7 +70,7 @@ def setUp(self):
 
     @patch("streamlit.watcher.local_sources_watcher.PathWatcher")
     def test_just_script(self, fob):
-        lso = local_sources_watcher.LocalSourcesWatcher(SCRIPT_PATH)
+        lso = local_sources_watcher.LocalSourcesWatcher(PagesManager(SCRIPT_PATH))
```

Comment:
nit: not quite related to this change, but there variable used here is frequently `lso` (not sure if `LocalSourcesWatcher` previously had a different name and this variable stayed the same when class name was changed by a search and replace) when `lsw` would be clearer